codebase/composer.json

```diff
@@ -35,6 +35,7 @@
         "composer/installers": "^1.9",
         "cweagans/composer-patches": "^1.7",
         "drupal/admin_toolbar": "^3.1",
+        "drupal/advanced_search": "^2.0",
```

Comment:
Module has changed from `islandora/advanced_search` to `drupal/advanced_search`. Can you check?